include/envoy/server/admin.h

```diff
@@ -124,6 +124,7 @@ class Admin {
   virtual void startHttpListener(const std::string& access_log_path_,
                                  const std::string& address_out_path,
                                  Network::Address::InstanceConstSharedPtr address,
+                                 Network::Socket::OptionsSharedPtr socketOptions,
```

Comment:
nit: `const Network::Socket::OptionsSharedPtr& socket_options` here and elsewhere